src/Models/News/News.php

```diff
@@ -0,0 +1,37 @@
+<?php
+declare(strict_types=1);
+
+namespace Engelsystem\Models\News;
+
+use Illuminate\Database\Eloquent\Model;
+use Engelsystem\Models\User\User;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+/**
+ * This class represents a news item.
+ */
+class News extends Model
+{
+    protected $casts = [
+        'is_meeting' => 'boolean',
+    ];
+
+    protected $attributes = [
+        'is_meeting' => false,
```

Comment:
What?

Reply:
What what? :)

Reply:
Why is it required to set the value here?

Reply:
It's the default value when you create a new instance using the model functions